salt/transport/__init__.py

```diff
@@ -308,3 +308,36 @@ def send(self, load, tries=3, timeout=60):
             return self._uncrypted_transfer(load, tries, timeout)
         else:  # for just about everything else
             return self._crypted_transfer(load, tries, timeout)
+
+class LocalChannel(Channel):
+    """
+    Local channel for testing purposes
+    """
```

Comment:
No double-quotes, please. See the convention guide for additional details. Thanks! http://docs.saltstack.com/en/latest/topics/development/conventions/style.html